extensions/2.0/Khronos/KHR_audio_graph/README.md

```diff
@@ -0,0 +1,1279 @@
+
+
+<table>
+  <tr>
+   <td>
+   </td>
+   <td><strong>Type</strong>
+   </td>
+   <td><strong>Description</strong>
+   </td>
+   <td><strong>Required</strong>
+   </td>
+   <td><strong>Notes</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>output channels
+   </td>
+   <td>integer
+   </td>
+   <td>Number of channels in output audio.
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>channel interpretation
+   </td>
+   <td>string
+   </td>
+   <td>speakers, discrete
+<p>
+Channel ordering for speaker channel interpretation are captured <a href="https://webaudio.github.io/web-audio-api/#ChannelOrdering">here</a>. When the number of channels do not match any of the basic speaker layouts, use discrete as it maps inputs to channels.
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+  </tr>
+</table>
+
+
+
+### 6.6 Channel mixer node (1 input / 1 output)
+
+Up-mixing refers to the process of taking a stream with a smaller number of channels and converting it to a stream with a larger number of channels. Down-mixing refers to the process of taking a stream with a larger number of channels and converting it to a stream with a smaller number of channels. Channel mixer node should ideally use these [mixing rules](https://webaudio.github.io/web-audio-api/#mixing-rules).
+
+
+<table>
+  <tr>
+   <td>
+   </td>
+   <td><strong>Type</strong>
+   </td>
+   <td><strong>Description</strong>
+   </td>
+   <td><strong>Required</strong>
+   </td>
+   <td><strong>Notes</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>input channels
+   </td>
+   <td>integer
+   </td>
+   <td>Number of channels in input audio.
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>output channels
+   </td>
+   <td>integer
+   </td>
+   <td>Number of channels in output audio.
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>channel interpretation
+   </td>
+   <td>string
+   </td>
+   <td>speakers, discrete
+<p>
+Speakers use up-mix / down-mix equations. In cases where the number of channels do not match any of the basic speaker layouts, use "discrete". "Discrete" up-mix by filling channels until they run out then zero out remaining channels; down-mix by filling as many channels as possible, then dropping remaining channels.
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+  </tr>
+</table>
+
+
+
+### 6.7 Audio mixer node (N inputs / 1 output)
+
+Use the Audio mixer node to combine the output from multiple audio sources. Number of channels should be the same across all inputs.
+
+[no properties for this node]
+
+
+### 6.8 Filter node (1 input / 1 output)
+
+Use the Audio Mixer node to combine the output from multiple audio sources. A filter node always has exactly one input and one output.
```

Comment:
This looks to be partially copied from the previous item by mistake.

Reply:
Updated.